utils/Dockerfile-kwiver-dump-klv

```diff
@@ -0,0 +1,14 @@
+# An image for extracting the KLV metadata from FMV datasets usign Kwiver
+# This accepts a movie file on stdin and produces the KLV metadata on stdout
+# This is pushed to https://hub.docker.com/repository/docker/banesullivan/kwiver/tags
+#  under the tage `dump-klv`
+#
+# Usage:
+#   cat video.mpg | docker run -i banesullivan/kwiver:dump-klv > video.klv
+FROM banesullivan/kwiver:latest
+
+# NOTE: vodeo file extension does not matter.
+CMD bash -c '\
+  cat > fmv.video && \
+  /kwiver/build/bin/kwiver dump-klv fmv.video \
+'
```

Comment:
This is a Dokerfile for extracting KLV metadata from any FMV dataset. I wasn't sure where the best place to keep this Dockerfile would be, so I posted here and pushed the image to DockerHub [here](https://hub.docker.com/repository/docker/banesullivan/kwiver/tags?page=1) under the `dump-klv` tag. Perhaps this should migrate over to Kwiver down the road.

Also, note that we just pull the image and don't actually use this Dockerfile anywhere.